File: 00 BASICS/ARRAYS/Array.py
# INITIALIZATION
myArray = ["Tim", "Julia", "Hans"]

# READ
print(myArray)										# print complete Array
print(myArray[2])									# print one element of the Array (2nd index)

# CREATE
myArray.insert(1, "Lola")								# Insert into specific position
myArray.append("Peter")									# Append to the end of the array

# UPDATE
myArray[0] = "Zoe"									# Updates Value of an Element of the Array

# ITERATE BY INDEX
for i in range(len(myArray)):								
	print(myArray[i])
	
# ITERATE BY ELEMENT
for name in myArray:									
	print(name)

# SORT
myArray.sort()										# ASC, myArray.sort(reverse=True) for desc

# REVERSE
myArray.reverse()

# DELETE element
myArray.remove("Julia")									# Remove Element by Value
del myArray[2]										# Remove Element by Index

# DELETE all
myArray.clear()										# Removes everything

File: 00 BASICS/ARRAYS/Array.js
// INITIALIZATION
myArray = ["Tim", "Julia", "Hans"];

// READ
console.log(myArray);                           // prints complete Array
console.log(myArray[1])                         // prints one element of the array

// CREATE
myArray.push("Lola");                           // Append to the end of the Array
myArray.splice(1, 0, "Peter");                  // Insert into (Position, Delete Count, Element)

// UPDATE
myArray[0] = "Zoe"                              // Updates Value of Element by Index

// ITERATATION BY INDEX
for (let i = 0; i < myArray.length; i++) {      // Iterate by index
    console.log(myArray[i]);
}

// ITERATION BY ELEMENT
for (let name of myArray) {                     // Iterate by Element
    console.log(name);
}

// SORT
myArray.sort()                                  // ASC, myArray.sort().reverse() for desc

// REVERSE
myArray.reverse()                               // Reverses the order of the Array

// DELETE element
myArray.filter(e => e !== "Julia")              // Remove by Value with Arrow Function
myArray.splice(3, 1)                            // Remove by Index, Delete Count

// DELETE all
myArray.length = 0                              // Removes everything
